apps/src/lib/node/ledger/tendermint_node.rs

```diff
@@ -190,6 +190,26 @@ pub fn reset(tendermint_dir: impl AsRef<Path>) -> Result<()> {
     Ok(())
 }
 
+pub fn rollback(tendermint_dir: impl AsRef<Path>) -> Result<()> {
+    let tendermint_path = from_env_or_default()?;
+    let tendermint_dir = tendermint_dir.as_ref().to_string_lossy();
+
+    // Rollback tendermint state
+    std::process::Command::new(tendermint_path)
```

Comment:
Are there docs we can link to that explain how Tendermint rollback behaves?

Reply:
I wasn't able to find proper documentation for this. We could link [this](https://github.com/tendermint/tendermint/blob/64747b2b184184ecba4f4bffc54ffbcb47cfbcb0/cmd/tendermint/commands/rollback.go) file which contains the help message, or simply include it in the help message of the Namada command

Reply:
let's include the link to the code at least